examples/simple.mjs

```diff
@@ -0,0 +1,22 @@
+import * as mg from '../modules/mgclient.mjs';
+
+mg.factory().then(async (instance) => {
+    console.log("Hello world, jsmgclient");
+    mg.init(instance);
+    console.log("version is " + mg.clientVersion());
+    let mgClient = await mg.MgClient.connect("0.0.0.0", 7687);
+    if (mgClient == null) {
+        throw "Could not connect to memgraph!";
+    }
+    let result = await mgClient.execute("CREATE (n:Person { name: 'Kostas', Age: '27' })");
+    await mgClient.discardAll();
+    result = await mgClient.execute("MATCH (n) return n");
+    let totalRows = 0;
+    let row;
+    while (row = await mgClient.fetchOne()) {
+        ++totalRows;
+    }
```

Comment:
Consider adding higher-level methods like `executeAndFetchAll` something which is a bit easier to use from the user perspective 😄

Reply:
These are low-level API's that a small expressive library would be built ontop. I rather not pollute the "bare bone" communication with WASM mgclient. I would provide one more wrapper that effectively returns a javascript list instead of an MgList and so on but this should not be a part of this PR. I can expand more on this on the daily